Lib/sqlite3/__init__.py

```diff
@@ -55,3 +55,11 @@
 """
 
 from sqlite3.dbapi2 import *
+from sqlite3.dbapi2 import _deprecated_version_info, _deprecated_version
+from warnings import warn
```

Comment:
Importing `warnings` imposes a performance penalty, but it's only needed if somebody tries to access one of the deprecated names in the module namespace. You should move the import inside the `if name in deprecated_names` block instead